pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -1093,6 +1093,14 @@ func Convert_v1_VirtualMachine_To_api_Domain(vmi *v1.VirtualMachineInstance, dom
 		})
 	}
 
+	if needsSCSIControler(vmi) {
+		domain.Spec.Devices.Controllers = append(domain.Spec.Devices.Controllers, Controller{
+			Type:  "scsi",
+			Index: "0",
+			Model: "virtio-scsi",
```

Comment:
do we need to take into account iothreads on this controller?

Reply:
I don't think so. At least not now. This controller was always added like this by libvirt by default if a scsi controller was required. I only add it to ensure that we always get the right model, since we now use latest libvirt/qemu releases instead of rhel-av based releases.